### PR TITLE
chore(SRVKP-4523): add tekton resolver deadlock panels/metric

### PR DESCRIPTION
**Pull request by gabemontero**
Signed-off-by: Gabe Montero <gmontero@redhat.com>

@amisstea PTAL

@enarha @savitaashture @divyansh42 FYI

deadlock-resolutionreq-panel

rh-pre-commit.version: 2.3.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

**Review by yftacherzog — COMMENTED** [review]
lgtm for metrics and their unit tests

**Comment by gabemontero** [comment]
bump @amisstea - thanks

File: rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml [hunk 1]
```diff
@@ -92,3 +92,18 @@ spec:
...
+            severity: critical
+            slo: "true"
+          annotations:
+            summary: >-
+              Tekton resolver controller appears to have stopped processing active resolutionrequests which have not been started yet.
+            description: >-
+              Tekton resolver controller on cluster {{ $labels.source_cluster }} has appeared deadlocked on {{ $value }} resolutionrequests.
+            alert_team_handle: <!subteam^S04PYECHCCU>
+            team: pipelines
+            runbook_url: TBD
```
Comment (amisstea) [inline rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:109]: LGTM. Just don't forget to return and add an SOP!

**Review by amisstea — APPROVED** [review]

File: rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml (hunk 1 above)
Comment (gabemontero) [reply rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:109]: the SOP is getting worked over in gitlab and MR 112711 @amisstea  .... if you mean something else though, sorry I am missing it .... if I a mssing it, please clarify - thanks.

Comment (amisstea) [reply rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:109]: That's perfect. Ty!

Comment (gabemontero) [reply rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:109]: likewise thanks

**Pull request merged**